PRODUCTIONISATION.md

```diff
@@ -97,6 +97,8 @@ Ensuring that your project adheres to the following RFCs is a good first step as
 * [ ] 7.2 Alerts sent to relevant instant chat channel
 * [ ] 7.3 Alerts sent to relevant team email address
 * [ ] 7.4 UUID of request is sent as additional user data
+* [ ] 7.5 No unresolved exceptions in the past week  
```

Comment:
```suggestion
* [ ] 7.5 reported exceptions are resolved within a week
```
This orientates the item to be more around a process + discipline, which is more inline with what the description is talking about.